openedx/core/djangoapps/heartbeat/runchecks.py

```diff
@@ -0,0 +1,43 @@
+from importlib import import_module
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from .defaults import (HEARTBEAT_DEFAULT_CHECKS,
+                       HEARTBEAT_EXTENDED_DEFAULT_CHECKS,
+                       HEARTBEAT_CELERY_TIMEOUT)
```

Comment:
This is the argument for using an IDE. I didn't have these issues in PHP, and when I wrote Java I used Eclipse